test/io/readline/readStridedArr.chpl

```diff
@@ -3,18 +3,20 @@ use IO;
 var A: [1..20 by 2] uint(8);
 var numread: int;
 
-readline(A, numread);
+numread = readLine(A);
 writeln("Got ", numread, ":", A[1..2*numread by 2]);
-readline(A, numread);
+numread = readLine(A);
 writeln("Got ", numread, ":", A[1..2*numread by 2]);
 
 // read past EOL
-while (readline(A, numread, amount=1)) {
+numread = readLine(A, maxSize=1);
+while (numread > 0) {
```

Comment:
Not worth changing, but I recently wrote a number of AoC codes using a style like this before realizing I could just write:

```chapel
while (readLine(A, maxSize=1) > 0) {
  ...
}
```

...which is so much nicer without the duplication!  I'm still kicking myself.